### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,11 +74,8 @@ The libpostal types seems nice (and made by brighter people than us):
 ### Output schema
 :construction:
 
-## Future
-:construction:
-
 ## Dataset quality test
-:construction: how we plan to ensure the quality of the released dataset :construction:
+:construction: how we plan to ensure the quality of the released dataset. Contributions welcomed in issue #4 :construction:
```
Comment (amatissart) [inline README.md]: Add a link to #4, maybe.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
+You will need
+* rust (`curl https://sh.rustup.rs -sSf | sh`)
+* GEOS (`apt-get install libgeos-dev`)
+
+Then, build cosmogony: `cargo build --release`
+
+You can now grab some OSM pbf and extract your geographic zones:
 
-## Use cases
+`./target/release/cosmogony -i /path/to/your/file.osm.pbf --libpostal /path/to/libpostal/rules/folder/ -o /path/for/output/file`
```
Comment (antoine-de) [inline README.md]: it's better to use `cargo run --release`

`cargo run --release -- -i /path/to/your/file.osm.pbf --libpostal /path/to/libpostal/rules/folder/ -o /path/for/output/file`

File: README.md [hunk 3]
```diff
@@ -1,22 +1,37 @@
...
+
+Check out cosmogony help for more options: `./target/release/cosmogony -h`
```
Comment (antoine-de) [inline README.md]: same here

File: README.md [hunk 4]
```diff
@@ -1,22 +1,37 @@
...
 
-We need a structured admin hierarchy to easily know that [paris](https://www.openstreetmap.org/relation/7444) is `city` in the `state` [ile de france](https://www.openstreetmap.org/relation/8649) in the `country` [france](https://www.openstreetmap.org/relation/2202162).
+## Use
 
-OSM administrative regions may not be mapped with the same precision all over the earth but the data is easy to update and the update will benefit the community.
+### Get data
+:construction: We may provide direct data download in the future. For now, you have to extract your geographic regions by yourself :construction:
 
-We do not forbid ourself however to use other data sources (with compliant license), but we don't want `cosmogony` to be too complex and we do not aim to recreate the great [WhosOnFirst](https://www.whosonfirst.org/) ([see below](#See-also))
+#### Build
```
Comment (antoine-de) [inline README.md]: I think you should add a note telling the update the submodules

```md
update the git submodules:
`git submodule update --init`
```

File: README.md [hunk 5]
```diff
@@ -1,22 +1,37 @@
...
+You will need
+* rust (`curl https://sh.rustup.rs -sSf | sh`)
+* GEOS (`apt-get install libgeos-dev`)
+
+Then, build cosmogony: `cargo build --release`
+
```
Comment (antoine-de) [inline README.md]: I would put a `Use` subsection

File: README.md [hunk 6]
```diff
@@ -1,22 +1,37 @@
...
+### Get data
+:construction: We may provide direct data download in the future. For now, you have to extract your geographic regions by yourself :construction:
 
-We do not forbid ourself however to use other data sources (with compliant license), but we don't want `cosmogony` to be too complex and we do not aim to recreate the great [WhosOnFirst](https://www.whosonfirst.org/) ([see below](#See-also))
+#### Build
+You will need
+* rust (`curl https://sh.rustup.rs -sSf | sh`)
+* GEOS (`apt-get install libgeos-dev`)
+
+Then, build cosmogony: `cargo build --release`
```
Comment (antoine-de) [inline README.md]: maybe add a section on the unit tests ?

Comment (nlehuby) [reply README.md]: you don't actually need to run the tests if you just want to use cosmogony. This may be in a `contribute` section, what do you think ?

Comment (antoine-de) [reply README.md]: yes a contribute section is nice

**Pull request merged**